x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_utils/timeseriesexplorer_utils.js

```diff
@@ -153,6 +153,9 @@ export function processDataForFocusAnomalies(
         chartPoint.anomalyScore = recordScore;
         chartPoint.function = record.function;
 
+        if (chartPoint.value === null) {
```

Comment:
The check for `actual` right below makes me think that it's possible for `actual` to be `undefined`. 
Might be good to move this into that if block to ensure `actual` is defined before we set `chartPoint.value` to it.

Reply:
Agree with @alvarezmelissa87. It should probably go into both parts of the `if` block below in fact, including the part with `if (causes.length === 1) ` to cover off the case where the gap is for a population detector with `by` and `over` fields, with a single `cause` (in which case there is no `actual` at the top level of the anomaly record.

Reply:
Thanks all. Ya'll right, I have updated it in  https://github.com/elastic/kibana/pull/66071/commits/64962d1be4287382747a3aec393aef7c61b8b7c9.